experimental/packages/api-logs/src/types/Event.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface Event {
```

Comment:
Events do not have span context properties?

Reply:
I have added the context properties. @scheler just double checking that the intent was to be able to add them manually (in addition to the `includeTraceContext` config for automatic inclusion).

Reply:
I'm not for or against manually including them, just thought it was odd not to be consistent

Reply:
Events do need to have span context and the SDK should add it to the event when `includeTraceContext` is on. The `includeTraceContext` option is added to support the case of not tagging span context of an unrelated in-progress span in an event in the same execution context. For eg., if a user clicks on a button in the browser while a span is in progress, you may not want to include this span's context in the user click event, and in this case the event needs to be created from a `logger` that has `includeTraceContext` off.

Reply:
After a conversation with @scheler, I have removed the `includeTraceContext` option for now, as we could not think of a use case for it.